### PR TITLE
Add lint error checking for scss files

### LINKED ISSUE #687: Add lint checks for scss files
Currently, *.scss files are not being checked for having lint issues. It would be beneficial to have automatic lint checks and fixes for *.scss files in order to have consistency across styling files.

### PR DESCRIPTION
**Pull request by ViliusBa**
With this PR *.scss files will be checked for lint to warn if standard scss guidelines are broken in `tree-widget` or `property-grid`.

Note that, a VS Code extension [stylelint](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint) will be needed to enable highlighting within the IDE.

### CODE REVIEW

**Review by aruniverse — APPROVED** [review]
Missing rush change

File: packages/itwin/tree-widget/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "copy:esm": "cpx \"./src/**/*.{scss,json}\" ./lib/esm",
     "cover": "nyc npm run test",
     "extract-api": "betools extract-api --entry=tree-widget-react",
-    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2 && npx stylelint \"**/*.scss\"",
```
Comment (saskliutas) [inline packages/itwin/tree-widget/package.json:36]: `npx` shouldn't be needed.

```suggestion
    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2 && stylelint \"./src/**/*.scss\"",
```

Maybe separate scripts for `eslint` and `stylelint`?
```
"lint": "npm run lint:eslint && npm run lint:stylelint",
"lint:eslint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
"lint:stylelint": "stylelint \"./src/**/*.scss\"",
```

File: packages/itwin/property-grid/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
     "cover": "nyc npm run test",
     "extract-api": "betools extract-api --entry=property-grid-react",
-    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2 && npx stylelint \"**/*.scss\"",
```
Comment (saskliutas) [inline packages/itwin/property-grid/package.json:37]: `npx` shouldn't be needed.
```suggestion
    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2 && stylelint \"./src/**/*.scss\"",
```

Maybe separate scripts for `eslint` and `stylelint`?
```
"lint": "npm run lint:eslint && npm run lint:stylelint",
"lint:eslint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
"lint:stylelint": "stylelint \"./src/**/*.scss\"",
```

**Review by saskliutas — CHANGES_REQUESTED** [review]

File: packages/itwin/tree-widget/src/components/trees/VisibilityTreeBase.scss [hunk 3]
```diff
@@ -58,10 +60,8 @@
           }
         }
       }
-    }
 
-    &.enlarge {
-      .core-tree-node {
+      &.enlarge {
```
Comment (saskliutas) [inline packages/itwin/tree-widget/src/components/trees/VisibilityTreeBase.scss:64]: Looks like this is a duplicate of the one on line 8

**Review by saskliutas — CHANGES_REQUESTED** [review]

**Review by saskliutas — APPROVED** [review]

**Pull request merged**